lms/static/js/ccx/schedule.js

```diff
@@ -139,7 +139,7 @@ var edx = edx || {};
               return !node.hidden;});
             this.$el.html(schedule_template({chapters: this.showing}));
             $('table.ccx-schedule .sequential,.vertical').hide();
-            $('table.ccx-schedule .toggle-collapse').on('click', this.toggle_collapse);
+            $('table.ccx-schedule .unit a').on('click', this.toggle_collapse);
```

Comment:
We prefer to target classnames rather than bare elements. Would you please provide a classname for this `a` and then use that classname here in place of it?